docs/admin/admission-controllers.md

```diff
@@ -426,6 +450,11 @@ We strongly recommend using this plug-in if you intend to make use of Kubernetes
 ## Is there a recommended set of plug-ins to use?
 
 Yes.
+For Kubernetes >= 1.9.0, we strongly recommend running the following set of admission control plug-ins (order matters):
+
+```shell
+--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,GenericAdmissionWebhook,ResourceQuota,DefaultTolerationSeconds,MutatingAdmissionWebhook
```

Comment:
@kubernetes/sig-auth-api-reviews who's ready to add PSP to the list?

Reply:
Others to add:
 1. OwnerReferencesPermissionEnforcement
 2. NodeRestriction
 3. PodNodeSelector
 4. PodTolerationRestriction - @kubernetes/sig-scheduling-api-reviews the annotation have been alpha for a while, but the taints are not alpha.  Ready for this?

Reply:
it's called ValidatingAdmissionWebhook

Reply:
Let's add those in a followup PR.